htcmock/action.yml

```diff
@@ -3,7 +3,8 @@ description: "Action running an HTC Mock session from a Kubernetes Job"
 inputs:
   armonik-core-version:
     description: "Version of ArmoniK.Core to use"
-    required: true
+    required: false
+    default: "0.27.0"
```

Comment:
I would highly recommend to keep required to avoid having a dependency between the action and the actual infra.